sql/updates/world/master/2021_12_19_00_world.sql

```diff
@@ -0,0 +1,703 @@
+(179262, 14, 0, 0, 2104, 41079);
+
+DELETE FROM `creature_template_movement` WHERE `CreatureId`=179262;
+INSERT INTO `creature_template_movement` (`CreatureId`, `Ground`, `Swim`, `Flight`, `Rooted`, `Chase`, `Random`) VALUES 
+(179262, 1, 0, 1, 0, 0, 0);
+
+ -- Anduin Wrynn
+DELETE FROM `creature_template` WHERE `entry`=178072;
+INSERT INTO `creature_template` (`entry`, `name`, `femaleName`, `subname`, `TitleAlt`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `HealthScalingExpansion`, `RequiredExpansion`, `VignetteID`, `faction`, `npcflag`, `speed_walk`, `speed_run`, `scale`, `rank`, `dmgschool`, `BaseAttackTime`, `RangeAttackTime`, `BaseVariance`, `RangeVariance`, `unit_class`, `unit_flags`, `unit_flags2`, `unit_flags3`, `dynamicflags`, `family`, `trainer_class`, `type`, `type_flags`, `type_flags2`, `lootid`, `pickpocketloot`, `skinloot`, `VehicleId`, `mingold`, `maxgold`, `AIName`, `MovementType`, `HoverHeight`, `HealthModifier`, `HealthModifierExtra`, `ManaModifier`, `ManaModifierExtra`, `ArmorModifier`, `DamageModifier`, `ExperienceModifier`, `RacialLeader`, `movementId`, `CreatureDifficultyID`, `WidgetSetID`, `WidgetSetUnitConditionID`, `RegenHealth`, `mechanic_immune_mask`, `spell_school_immune_mask`, `flags_extra`, `ScriptName`, `VerifiedBuild`) VALUES 
+(178072, 'Anduin Wrynn', NULL, NULL, NULL, NULL, 0, 62, 62, 0, 0, 0, 14, 35184372088832, 1, 1.14286, 1, 1, 0, 2000, 0, 1, 1, 2, 32784, 71303168, 98336, 0, 0, 0, 0, 0, 0, 0, 0, 0, 584, 0, 0, '', 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 'npc_sylvanas_windrunner_anduin', 0);
+
+DELETE FROM `creature_template_addon` WHERE `entry`=178072;
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `aiAnimKit`, `movementAnimKit`, `meleeAnimKit`, `auras`) VALUES
+(178072, 0, 0, 0, 1, 0, 0, 0, 0, '358166');
+
+DELETE FROM `creature_template_scaling` WHERE `Entry`=178072;
+INSERT INTO `creature_template_scaling` (`Entry`, `DifficultyID`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `ContentTuningID`, `VerifiedBuild`) VALUES
+(178072, 14, 0, 0, 2104, 41079);
+
+DELETE FROM `creature_template_movement` WHERE `CreatureId`=178072;
+INSERT INTO `creature_template_movement` (`CreatureId`, `Ground`, `Swim`, `Flight`, `Rooted`, `Chase`, `Random`) VALUES 
+(178072, 1, 0, 1, 0, 0, 0);
+ 
+DELETE FROM `creature` WHERE `guid`=@CGUID+5;
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnDifficulties`, `phaseUseFlags`, `PhaseId`, `PhaseGroup`, `terrainSwapMap`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `unit_flags2`, `unit_flags3`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES 
+(@CGUID+5, 178072, 2450, 13561, 13655, '14', 0, 0, 0, -1, 0, 1, -249.161, -1271.62, 5667.08, 4.56334, 7200, 0, 0, 23058000, 2568, 0, 0, 0, 0, 0, 0, '', 41079);
```

Comment:
I normally keep these separated because it's easier for me to track what I modified for each table for the given unique ID. If it were too many stuff, like mass-parsing a whole zone, I would do it.